Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -91,8 +91,7 @@
           StandardOutputImportance="Low" />
     <!-- Here we link to Node.js Extensions as an npm package, which makes it available to the consumer project under
          its package name "nodejs-extensions". -->
-    <Exec Command="pnpm link $(NodeJsExtensionsNpmPackageSourcePath)"
-          StandardOutputImportance="Low" />
```

Comment:
Why remove this?

Reply:
If you don't set up the project that consumes _Lombiq.NodeJs.Extensions_ correctly (e.g. missing or incorrect _package.json_ file), then it will fail at this exec, without any info in the build log. So it's nice to strategically enable this to help out clueless people (talking about myself 🙃).

Reply:
What exactly needs to be set up correctly?

Reply:
[This.](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/158/files#diff-cc6f98dbcf68f7691f4dbf535e26e1d1a8963d9777b8f680dfac9b5bd8babebb) Otherwise that step fails and it says:

>pnpm link ./node_modules/.nx
>ΓÇëERR_PNPM_NO_IMPORTER_MANIFEST_FOUNDΓÇë No package.json (or package.yaml, or package.json5) was found in "E:\aux_c\repositories\orchard-core\Open-Source-Orchard-Core-Extensions2\NuGetTest\src\Lombiq.OSOCE.NuGet.Web".
> *Error MSB3073 : The command "pnpm link ./node_modules/.nx" exited with code 1.*

If you don't remove `StandardOutputImportance="Low"`, then it just says the last line without any context which is very unhelpful.

Reply:
Sounds good.